src/artm/messages.proto

```diff
@@ -360,8 +360,9 @@ message TopTokensScore {
   repeated int32 topic_index = 3;
   repeated string token = 4;
   repeated float weight = 5;
-  repeated float coherence = 6;
-  optional float average_coherence = 7;
+  repeated string coherence_topic_name = 6;
```

Comment:
@ofrei Is it OK to add some backward incompatibility to `TopTokensScore` protobuf message?
Or is it better to just add new field to the end of message?